src/Sylius/Bundle/ApiBundle/DataTransformer/ChannelPricingOutputDataTransformer.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Sylius\Bundle\ApiBundle\DataTransformer;
+
+use ApiPlatform\Core\DataTransformer\DataTransformerInterface;
+use Sylius\Bundle\ApiBundle\DTO\ChannelPricingOutput;
+use Sylius\Bundle\ApiBundle\Helpers\PriceTransformer;
+use Sylius\Component\Core\Model\ChannelPricingInterface;
+
+class ChannelPricingOutputDataTransformer implements DataTransformerInterface
+{
+    /**
+     * @param ChannelPricingInterface $channelPricing
+     * @param string $to
+     * @param array $context
+     * @return object|ChannelPricingOutput
+     */
+    public function transform($channelPricing, string $to, array $context = [])
+    {
+        $dto = new ChannelPricingOutput();
+        $dto->id = $channelPricing->getId();
+        $dto->channelCode = $channelPricing->getChannelCode();
+        $dto->price = PriceTransformer::centimesToUnitsForNullableFields($channelPricing->getPrice());
+        $dto->originalPrice = PriceTransformer::centimesToUnitsForNullableFields($channelPricing->getOriginalPrice());
+
+        return $dto;
+    }
+
+    public function supportsTransformation($data, string $to, array $context = []): bool
+    {
+        return $to === ChannelPricingOutput::class;
+    }
```

Comment:
We need to ensure that DataTransformer working on correct object, so due to the fact, we have to implement supportsTransformation with correct validation
```suggestion
    public function supportsTransformation($data, string $to, array $context = []): bool
    {
        return $to === ChannelPricingOutput::class && $data instanceof ChannelPricingInterface;
    }
```